specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-05-01-preview/appplatform.json

```diff
@@ -11051,6 +11712,45 @@
       },
       "x-ms-discriminator-value": "Weekly"
     },
+    "ConfigServerResourceRequests": {
+      "description": "Resource request payload of Config Server, only available in Enterprise Plan.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "cpu": {
+          "description": "CPU allocated to each Config Server instance",
+          "type": "string",
+          "readOnly": true
+        },
+        "memory": {
+          "description": "Memory allocated to each Config Server instance",
+          "type": "string",
+          "readOnly": true
+        },
+        "instanceCount": {
+          "format": "int32",
+          "description": "Instance count of the Config Server",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "ConfigServerInstance": {
+      "description": "Collection of instances belong to the Config Server, only available in Enterprise Plan.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the Config Server instance",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
```

Comment:
is it enumerated?

Reply:
We have several existing APIs providing `status` for instance, which are `string` type, to keep consistency between APIs, `string` is used here as well.